### DISCUSSION TITLE
I have installed using a makefile but one of the containers is not running, namely attendaze_worker with the following error, please help

### DISCUSSION THREAD

**Discussion by eldyyuda** [Attendize/Attendize#1068] [Q&A]
![Error container](https://user-images.githubusercontent.com/108863543/201807280-dbea1edd-bf92-4e68-8f29-f6e3a5d9d5ed.PNG)

![Docker-gui-attend](https://user-images.githubusercontent.com/108863543/201807330-a5bc864b-83de-44ec-b159-847477923a22.PNG)

![Screenshot (237)](https://user-images.githubusercontent.com/108863543/201808783-c806229a-1e3a-42d9-91ba-f6fd041d0428.png)

**Comment by storca** [comment]
Hi !

I'm running Docker locally and I'm not sure that the ```worker``` container is required for your Attendize setup to work properly.

On my local dev setup, only the ```db```, ```maildev``` and ```web``` containers are running and Attendize works.

I may be totally wrong tho, please correct me if so.
